src/safeposix/syscalls/net_calls.rs

```diff
@@ -1272,21 +1383,19 @@ impl Cage {
                         );
                     }
 
+                    //The destaddr's address family must match that of the
+                    //socket's address from 'fd'. Otherwise, a message can't be sent
                     if dest_addr.get_family() != sockhandle.domain as u16 {
                         return syscall_error(
                             Errno::EINVAL,
                             "sendto",
                             "An address with an invalid family for the given domain was specified",
                         );
                     }
-                    if (flags & !MSG_NOSIGNAL) != 0 {
```

Comment:
why this is removed

Reply:
Nick mentioned that at one point it was needed, but that is not the case anymore after we started using pipes